docs/modules/ROOT/pages/how-tos/exoscale/install.adoc

```diff
@@ -14,46 +14,110 @@ This how-to guide is still a work in progress and will change.
+
 == Prerequisites
+
 * Exoscale https://community.exoscale.com/documentation/iam/quick-start/#api-keys[API key]
-* `terraform`
-* `exo` >= v1.14.0 https://community.exoscale.com/documentation/tools/exoscale-command-line-interface[Exoscale CLI]
+* `docker`
+* `exo` >= v1.28.0 https://community.exoscale.com/documentation/tools/exoscale-command-line-interface[Exoscale CLI]
 * `jq`
+* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor] (version 4 or higher)
 * `md5sum`
 * `virt-edit`
 * `cpio`
+* `openshift-install` (direct download: https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-{ocp-minor-version}/openshift-install-linux.tar.gz[linux], https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-{ocp-minor-version}/openshift-install-mac.tar.gz[macOS])
 * Clone of the https://github.com/appuio/terraform-openshift4-exoscale[terraform-openshift4-exoscale] repository
 * https://community.exoscale.com/documentation/dns/quick-start/#subscribing-to-the-service[DNS subscription] activated in the Exoscale organisation
+* `vault` https://www.vaultproject.io/docs/commands[Vault CLI]
+* `curl`
+* `gzip`
 * A copy of the Ursula Debian package
 
+[WARNING]
+====
+Make sure the version of openshift-install and the rhcos image is the same, otherwise ignition will fail.
+====
+
 == Cluster Installation
 
-. Register the new OpenShift 4 cluster in Lieutenant: https://control.vshn.net/syn/lieutenantclusters
+Register the new OpenShift 4 cluster in Lieutenant.
 
-. Configure input
+.Lieutenant API endpoint
+****
+Use the following endpoint for Lieutenant:
+
+VSHN:: https://api.syn.vshn.net
+****
+
+=== Set up LDAP service
+
+. Create an LDAP service
 +
+Use https://control.vshn.net/vshn/services/_create to create a service.
+The name must contain the customer and the cluster name.
+And then put the LDAP service ID in the following variable:
++
+[source,console]
+----
+export LDAP_ID="Your_LDAP_ID_here"
+export LDAP_PASSWORD="Your_LDAP_pw_here"
+----
+
+=== Configure input
+
+.Access to various APIs
 [source,console]
 ----
 export EXOSCALE_ACCOUNT=<exoscale-account>
 export EXOSCALE_API_KEY=<exoscale-key>
 export EXOSCALE_API_SECRET=<exoscale-secret>
-export EXOSCALE_REGION=ch-dk-2
-export CLUSTER_ID=<cluster-name>
-export BASE_DOMAIN=<cluster-base-domain>
-export PULL_SECRET=<redhat-pull-secret> # From https://cloud.redhat.com/openshift/install/pull-secret
+export EXOSCALE_REGION=<exoscale-zone>
 
-export AWS_ACCESS_KEY_ID=${EXOSCALE_API_KEY}
-export AWS_SECRET_ACCESS_KEY=${EXOSCALE_API_SECRET}
-export AWS_S3_ENDPOINT="sos-${EXOSCALE_REGION}.exo.io"
+# From https://git.vshn.net/profile/personal_access_tokens
+export GITLAB_TOKEN=<gitlab-api-token>
+
+# For example: https://api.syn.vshn.net
+# IMPORTANT: do NOT add a trailing `/`. Commands below will fail.
+export COMMODORE_API_URL=<lieutenant-api-endpoint>
+# See https://wiki.vshn.net/x/ngMBCg#ClusterRegistryinLieutenantSynthesizeaCluster(synfection)-Preparation
```

Comment:
Not sure what was supposed to be explained in more detail in the synthesize guide in the wiki. I'd say we just remove this line.